README.md

```diff
@@ -688,6 +688,50 @@ Example ESLint configuration:
 }
 ```
 
+### [unicorn/template-indent]
+
+**This rule can be used with certain options.**
+
+This rule will automatically fix the indentation of multiline string templates, to keep them in alignment with the code they are found in. A configurable whitelist is used to ensure no whitespace-sensitive strings are edited.
+
+Prettier deals with:
+
+- html
+- css
+- graphql
+- markdown
+
+Using various tags, functions and comments. See: https://github.com/prettier/prettier/tree/main/src/language-js/embed
```

Comment:
I don’t think the link helps regular users.

Reply:
I added it so that if Prettier started supporting more formats we wouldn't have to continuously update the file

Reply:
Prettier could just as likely refactor the code so that the links stops working

Reply:
You'd fancy a permalink? 😁

Just kidding, will remove it